opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -352,8 +352,19 @@ message IntDataPoint {
 // NumberDataPoint is a single data point in a timeseries that describes the
 // time-varying value of a double metric.
 message NumberDataPoint {
-  // The set of labels that uniquely identify this timeseries.
-  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
+  // The set of key/value pairs that uniquely identify the timeseries from 
```

Comment:
Nit:  Should we call out error scenarios and what to do with competing types in same-label-key across different Metrics, or just but a TODO to call that out in the DataModel Spec?

Reply:
I am comfortable with a TODO to call this out in the data model spec. As it stands, this file could use a complete rewrite of its comments, but I'd rather discuss and debate what those comments should really say in the data model document, and then copy sections of the data model back into the .proto at the end.